lib/tools/transformer/expression_generator.dart

```diff
@@ -84,8 +84,7 @@ class ExpressionGenerator extends Transformer with ResolverTransformer {
         .toList();
 
     // Get all of the contents of templates in @Component(templateUrl:'...')
-    gatherReferencedUris(transform, resolver, options,
-        templatesOnly: true).then((templates) {
```

Comment:
Looks like that was the only call to `gatherReferencedUris`, I wonder if we can simplify the code this by removing templatesOnly as an option altogether. Not a prerequisite for merging this.